06.bitstream.syntax.md

```diff
@@ -85,18 +85,26 @@ in the [AOMedia metadata registry](https://aomediacodec.github.io/metadata-regis
 |     @@apply_units_resolution_log2                         | f(4)
 |     @@apply_horz_resolution                               | f(12)
 |     @@apply_vert_resolution                               | f(12)
+|     @@mono_chrome                                         | f(1)
+|     SubX = SubY = 0
+|     if ( !mono_chrome ) {
+|          subsampling_x                                    | f(1)
+|          SubX = subsampling_x
+|          subsampling_y                                    | f(1)
+|          SubY = subsampling_y
+|     }
 |     @@video_frame_characteristics_flag                    | f(1)
-|     if (video_frame_characteristics_flag)
-|         @@bit_depth                                       | f(3)
-|         BitDepth = bit_depth + 8
-|         @@mono_chrome                                     | f(1)
-|         SubX = SubY = 0
-|         if ( !mono_chrome ) {
-|             subsampling_x                                 | f(1)
-|             SubX = subsampling_x
-|             subsampling_y                                 | f(1)
-|             SubY = subsampling_y
-|         }
+|     if (video_frame_characteristics_flag) {
+|         @@bit_depth_minus8                                | f(3)
+|         BitDepth = bit_depth_minus8 + 8
+|         color_primaries                                   | f(8)
+|         transfer_characteristics                          | f(8)
+|         matrix_coefficients                               | f(8)
+|     }
+|     else {
+|         color_primaries = CP_UNSPECIFIED	 
```

Comment:
color_primaries = CP_UNSPECIFIED.

They are actually equal to the video frame color primaries.

Reply:
Removed CICP signaling for this PR as requested.  Will address comments in new PR.